lineman/spec/helpers/directive.js

```diff
@@ -0,0 +1,10 @@
+window.prepareDirective = function(spec, name, setupFn) {
+  return inject(function($compile, $rootScope) {
+    parent = $rootScope.$new();
+    if (typeof setupFn === 'function') { setupFn(parent); }
+    spec.$element = $compile(angular.element('<' + name + ' discussion="discussion"></' + name + '>'))(parent);
```

Comment:
This will need some further massaging as we apply it to more directives, but I'm happy with how it works now.